MH4U Dex/HomeViewController.m

```diff
@@ -0,0 +1,37 @@
+//
+//  HomeViewController.m
+//  MH4U Dex
+//
+//  Created by Joseph Goldberg on 2/5/15.
+//  Copyright (c) 2015 Joseph Goldberg. All rights reserved.
+//
+
+#import "HomeViewController.h"
+
+#import "MonsterEncyclopediaViewController.h"
+#import "ItemEncyclopediaTableViewController.h"
+#import "RegionEncyclopediaCollectionViewController.h"
+
+@interface HomeViewController ()
+
+@end
+
+@implementation HomeViewController
+
+- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender
+{
+    if ([segue.identifier isEqualToString:@"showMonsterEncyclopedia"]) {
```

Comment:
I think this would be nice, but I wasn't really sure on how to install this tool.  Maybe we can go over it later?  I think it would be nice to set aside some time to have all identifiers be storyboard-generated constants.